Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN apt-get update \
         python3 \
         rename \
         wget \
+        git \
```

Comment:
Where is this used?

Reply:
It's a remnant for the solution we adopted.
It can be removed, I simply forgot to remove it. 

Basically I taught missions makers how to upload missions to a git repo. When the server starts it pulls  the repo in the mpmissions folder. This way if the server is off (and in my community case will be most of the time) they can still upload it somewhere without having to turn it on and ftp the file to it.